ManualSetup/6_InsertData_USER_PERSONAL_RANK.sql

```diff
@@ -0,0 +1,101 @@
+INSERT INTO [USER_PERSONAL_RANK] (nRank, strPosition, nElmoUP, strElmoUserID, nElmoLoyaltyMonthly, nElmoCheck, nKarusUP, strKarusUserID, nKarusLoyaltyMonthly, nKarusCheck, nSalary, UpdateDate) VALUES
+(1, N'Gold Knight          ', 0, N'                     ', 0, 0, 0, N'                     ', 0, 0, 0, CAST(N'2012-11-08T07:33:00' AS DateTime)),
```

Comment:
USER_PERSONAL_RANK.strPosition, USER_PERSONAL_RANK.strElmoUserID, USER_PERSONAL_RANK.strKarusUserID should be trimmed like they were in my export.